pytorch_pfn_extras/profiler/__init__.py

```diff
@@ -0,0 +1,2 @@
+from pytorch_pfn_extras.profiler import record  # NOQA
+from pytorch_pfn_extras.profiler import time_summary  # NOQA
```

Comment:
I think `ppe.profiler.record.record` and `ppe.profiler.time_summary.time_summary` are a little bit long and annoying. How about revealing these attributes here directly?